scss/components/_soe_dm_brand_bar.scss

```diff
@@ -41,3 +41,35 @@
     }
   }
 }
+
+a {
+  font-weight: 400;
+}
+
```

Comment:
This has no other selectors so it sets "a" globally. I don't think this belongs here.

Reply:
it was in the injector: https://engineering.stanford.edu/admin/config/development/css-injector/edit/50

I just added what was in there.